tests/signInPageTests.spec.js

```diff
@@ -0,0 +1,37 @@
+import { test } from '@playwright/test'
+import HomePage from '../PageObjects/homePageFns'
+import Verify from '../Utils/Assertions/verify'
+import Click from '../Utils/Interaction/click'
+import pageSels from '../Utils/Selectors/pageSels'
+
+const { skipBtnText } = pageSels.signInPage
```

Comment:
change the class name it should have Pascal case

Reply:
Okay, will update that.